class.jetpack-options.php

```diff
@@ -123,15 +128,33 @@ public static function is_valid( $name, $group = null ) {
 	}
 
 	/**
+	 * Checks if an option must be saved for the whole network in WP Multisite
+	 *
+	 * @param string $option_name Option name. It must come _without_ jetpack_% prefix. The method will prefix the option name.
```

Comment:
May want to put backticks around the `jetpack_%` bit -- the underscores on the prior word can be a touch confusing for the eye.  Same remark for subsequent uses.